core/iwasm/libraries/lib-socket/src/wasi/wasi_socket_ext.c

```diff
@@ -504,7 +514,7 @@ getaddrinfo(const char *node, const char *service, const struct addrinfo *hints,
         return __WASI_ERRNO_SUCCESS;
     }
 
-    aibuf_res = calloc(1, addr_info_size * sizeof(struct aibuf));
+    aibuf_res = (aibuf *) calloc(1, addr_info_size * sizeof(struct aibuf));
```

Comment:
Should be `(struct aibuf *)`, or compilation error occurs in Ubuntu.

Reply:
this is done

Reply:
Yes, and please help format the coding style by using clang-format-12:
`clang-format-12 -i --style=file <this file>`